src/pages/PlanPage/subRoutes/CancelPlanPage/subRoutes/TeamPlanSpecialOffer/TeamPlanSpecialOffer.tsx

```diff
@@ -0,0 +1,45 @@
+import { useLayoutEffect } from 'react'
+
+import { useSetCrumbs } from 'pages/PlanPage/context'
+import A from 'ui/A'
+import Icon from 'ui/Icon'
+
+import TeamPlanCard from './TeamPlanCard'
+
+function TeamPlanSpecialOffer() {
+  const setCrumbs = useSetCrumbs()
+
+  useLayoutEffect(() => {
+    setCrumbs([
+      {
+        pageName: 'cancelOrgPlan',
+        text: 'Special offer',
+      },
+    ])
+  }, [setCrumbs])
+
+  return (
+    <div className="flex w-3/5 flex-col gap-4">
+      <div>
+        <h2 className="text-lg font-semibold">Alternative plan offer</h2>
+        <p>
+          We&apos;d like to introduce you to our lighter alternative - the Team
```

Comment:
Not sure if you saw in the bug ticket, we want to adjust this copy a bit too, see 2) here https://github.com/codecov/engineering-team/issues/790

Reply:
oops